key_value_store/key_value_service.cc

```diff
@@ -58,18 +79,85 @@ Status KeyValueStoreImpl::get(ServerContext* context,
 Status KeyValueStoreImpl::remove(ServerContext* context, 
                                      const RemoveRequest* request, RemoveReply* reply) {
   if (storage_.DeleteKey(request->key())) {
+    // If persistent mode is activated, append remove request into file.
+    if (persistent_mode_) {
+      Persistent_store("remove", request->key(), "");
+    }
     return Status::OK;
   }
   return Status(StatusCode::NOT_FOUND, "Failed to delete the key.");
 }
 
+// When instantiating a KeyValueStore class, load data from persistent_db_
+// if the file exists.
+void KeyValueStoreImpl::Persistent_load() {
+  if (persistent_db_.length() == 0) {
```

Comment:
Is persistent_db the filename or the db name? If it's the name of the file then it's a little unclear

Reply:
Fixed.